conductr-bundle-lib/src/main/java/com/typesafe/conductr/bundlelib/LocationService.java

```diff
@@ -51,7 +51,8 @@ public static URL getLookupUrl(String serviceName, URL fallback) throws Malforme
     }
 
     static HttpPayload createLookupPayload(String serviceLocator, String serviceName) throws MalformedURLException {
-        URL locatorUrl = new URL(serviceLocator + serviceName);
+        String serviceNameWithLeadingSlash = serviceName.startsWith("/") ? serviceName : "/" + serviceName;
```

Comment:
This here is the main change. Otherwise I've only added more tests to check service names without a leading slash.